Diijkstra_shortest_algo_in_python.py

```diff
@@ -0,0 +1,97 @@
+from collections import deque, namedtuple
+
+
+# we'll use infinity as a default distance to nodes.
+inf = float('inf')
+Edge = namedtuple('Edge', 'start, end, cost')
+
+
+def make_edge(start, end, cost=1):
+  return Edge(start, end, cost)

```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmppye1i2n8/Diijkstra_shortest_algo_in_python.py
+++ b/tmp/tmppye1i2n8/Diijkstra_shortest_algo_in_python.py
@@ -7,7 +7,7 @@
 
 
 def make_edge(start, end, cost=1):
-  return Edge(start, end, cost)
+    return Edge(start, end, cost)
 
 
 class Graph:
```